cypress/integration/login.js

```diff
@@ -8,20 +8,21 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    const roles = ['User', 'Team Lead', 'Manager', 'Accountant', 'Admin'];
+    const menu = ['1', '2', '5', '5', '6'];
+    roles.forEach((role, id) => {
+        it('Verify all user roles can log in and should see appropriate tabs', function() {

```

Comment:
It would be nice that each test would have specific name. You could do that by adding user role and menu items count in each test name